src/Serialization/upgrades/0.11.2.jl

```diff
@@ -0,0 +1,89 @@
+################################################################################
+# Upgrade Summary
+# Instances of the types in this union are not serialized with backref,
```

Comment:
Is this comment maybe in the wrong place? At least it is not clear what "this union" refers to here.

Shouldn't this comment rather start with something like "In this version, we changed how certain basic types are serialized, for a massive reduction in size of the serialized file"